src/app/components/activity/feed/devlog-post/text/text.vue

```diff
@@ -8,12 +8,14 @@
 		}"
 	>
 		<template v-if="isOpen">
-			<div class="page-cut" />
+			<div class="page-cut">
+				<div class="page-cut-content-placeholder" />
+			</div>
 
 			<app-content-viewer :source="post.article_content" />
 		</template>
 
-		<div class="page-cut">
+		<div class="page-cut page-cut-bottom">
```

Comment:
hyphenate `-page-cut-bottom` so that it doesn't conflict with page-cut in future